### ISSUE TITLE
Changes not updated when having multiple tabs/repos open

### ISSUE THREAD

**Issue by pr8x** [ForkIssues/TrackerWin#528]
I noticed that when having multiple tabs (i.e. repositories) open the changes/working tree list **of the active tab** sometimes doesn't update. When I switch to another tab and go back again it properly updates the list.

**Comment by DanPristupov** [comment]
Does this happen when you focus/activate Fork?

**Comment by pr8x** [comment]
Yes, when I focus the Fork and switch to changes the list doesn't update (sometimes).

**Comment by DanPristupov** [comment]
Hm... Is that possible that you opened some non-modal windows previously? For example File History.

**Comment by pr8x** [comment]
I usually don't use File History. In most scenarios I just Alt+Tab from Visual Studio to Fork in order to commit my changes. No other Fork window is open.

**Comment by DanPristupov** [comment]
How ofter do you see that problem? Once in a week/day/hour?

**Comment by pr8x** [comment]
I observed it 5 minutes ago :). Pretty frequently I would say, but not consistently.
